docs/modules/servers/pages/distributed/configure/jvm.adoc

```diff
@@ -103,3 +103,15 @@ Ex in `jvm.properties`
 # james.reactor.inputstream.prefetch=4
 ----
 
+== Relax mailbox name validation
+
+The property `james.relax.mailbox.name.validation` allows to accept `*` and `%` characters in mailbox name.
+
+Optional. Boolean. Default to true.
+
+Ex in `jvm.properties`
+----
+james.relax.mailbox.name.validation=false
+----
+To strictly validate `*` and `%` characters in the mailbox name.
```

Comment:
Please explain that % and * are ambiguous for the LIST / LSUB commands that interprete those as wildcard thus returning all mailboxes mating the pattern.